src/test/java/org/folio/rest/impl/AbstractRestTest.java

```diff
@@ -0,0 +1,107 @@
+package org.folio.rest.impl;
+
+import com.github.tomakehurst.wiremock.common.Slf4jNotifier;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.client.TenantClient;
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.PomReader;
+import org.folio.rest.tools.utils.NetworkUtils;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+
+import java.io.IOException;
+
+import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT;
+
+public abstract class AbstractRestTest {
+
+  protected static final String OKAPI_TENANT_HEADER = "x-okapi-tenant";
```

Comment:
RMB already has all  these variables defined. you can reuse them instead of re-defining
import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT; etc..

Reply:
done